api/src/main/java/com/jmanday/api/login/LoginService.kt

```diff
@@ -0,0 +1,14 @@
+package com.jmanday.api.login
+
+import retrofit2.http.Body
+import retrofit2.http.Headers
+import retrofit2.http.POST
+
+interface LoginService {
+    @Headers(
+        "x-api-key: Aj8vfl8S5QTEkhUD"
```

Comment:
You definitely don't want to put API keys in your source code and commit that to GitHub. Anyone who finds this can do bad with it. I recommend to put it in your `local.properties` file in Android Studio which is not pushed to Git by default. You can then read in properties from there as a build config field

I explain that here: https://www.youtube.com/watch?v=-2ckvIzs0nU

In the video, I use Gradle Groovy, but you can simply copy paste it into ChatGPT and ask it to convert to Gradle Kotlin. Then, after rebuilding, you can access the API key here with `BuildConfig.API_KEY`.

PS: It also makes sense to add this HTTP header with an interceptor. That's a class which will let you intercept every HTTP request made with a client to adjust it (like adding a header). Then you don't need to add this annotation to each and every request in the app.